packages/relay/tests/lib/hbarLimiter.spec.ts

```diff
@@ -31,16 +31,23 @@ describe('HBAR Rate Limiter', async function () {
   this.timeout(20000);
   let rateLimiter: HbarLimit;
   let currentDateNow: number;
-  const invalidDuration: number = 0;
+  let rateLimiterWithInvalidTotal: HbarLimit;
```

Comment:
`rateLimiterWithInvalidTotal` -> `rateLimiterWithEmptyBudget`

Reply:
I like the name! Thanks!